docs/pg-tde.md

```diff
@@ -194,4 +194,31 @@ To check if the data is encrypted, do the following:
     SELECT pg_tde_is_encrypted('my_encrypted_table');
     ```
 
-    The function returns `t` if the table is encrypted and `f` - if not.
\ No newline at end of file
+    The function returns `t` if the table is encrypted and `f` - if not.
+
+## Upgrade considerations
+
+Note that you cannot upgrade to `pg_tde` Beta with the UPDATE TO statement for the ALTER EXTENSION command. If you have previously installed `pg_tde` Alpha1 and wish to use the Beta version, you need to drop the extension and reenable the new version of it.
+
+Note that dropping the extension deletes all tables where it is enabled!
```

Comment:
Can we somehow make this stick out? Like bold or red or whatever applies to our documentation schema and rules?